src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -0,0 +1,12 @@
+#databaseChangeLog: []
+databaseChangeLog:
+  - include:
+      file: "/db/changelog/changes/1-create-info-phoneNumber-table-changelog.sql"
+  - include:
+      file: "/db/changelog/changes/2-create-info-role-table-changelog.sql"
+  - include:
```

Comment:
and maybe instead of writing the files one by one, you can use includeAll, then give the path for the changesets.

Reply:
I devised all creation process of tables written by XML for Liquibase.